boards/arc/emsdp/Kconfig.board

```diff
@@ -0,0 +1,11 @@
+# Kconfig - DesignWare ARC EM Software Development Platform board configuration
+#
+# Copyright (c) 2019 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+config BOARD_EMSDP
+	bool "EM Software Development Platform"
+	depends on SOC_ARC_EMSDP
+	help
+	  The ARC EM Software Development Platform (emsdp) is an FPGA based development platform intended to support ARC licenses in developing their software for the ARC EM processor family and ARC EM Subsystems. It has the support for ARC EM4, EM5D, EM6, EM7D, EM9D and EM11D processors. ARC EM Enhanced Security Package (ESP) and ARC EM Subsystems (DFSS, SCSS, DSS) are also supported.
```

Comment:
The help text should have newlines, see other multi-line help texts for how to do indentation correctly.

Reply:
already done.